tests/SideBySide/CancelTests.cs

```diff
@@ -228,6 +228,18 @@ public void DapperQueryMultiple()
 			}
 		}
 
+		[Fact]
+		public async Task CancelCompletedCommand()
+		{
+			using (var cmd = m_database.Connection.CreateCommand())
+			{
+				cmd.CommandText = @"select 1;";
+
+				using (await cmd.ExecuteReaderAsync().ConfigureAwait(false))
+					cmd.Cancel();
+			}
```

Comment:
The [Dispose](https://github.com/adampoit/MySqlConnector/blob/ef4f7eeb62e960ccb446a9a0ec5af12711931093/tests/SideBySide/CancelTests.cs#L17) method should be executing the second command, but perhaps it would be better to be explicit.